Mathlib/Data/Finset/Basic.lean

```diff
@@ -2710,7 +2710,9 @@ section Filter
 
 variable (p q : α → Prop) [DecidablePred p] [DecidablePred q] {s : Finset α}
 
-/-- `Finset.filter p s` is the set of elements of `s` that satisfy `p`. -/
+/-- `Finset.filter p s` is the set of elements of `s` that satisfy `p`.
+For example, one can use `s.filter (· ∈ t)` to get the intersection of `s` with a `t : Set α`
```

Comment:
I think this should be a new paragraph in the docs (single newlines mean nothing in many markdown parsers)
```suggestion
/-- `Finset.filter p s` is the set of elements of `s` that satisfy `p`.

For example, one can use `s.filter (· ∈ t)` to get the intersection of `s` with a `t : Set α`
```